wwr/about.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initialscale=1">
+  <title>About Us</title>
+  <meta name="description" content="Find out about our rafting water company">
+  <meta name="author" content="Josue Elias Leyva">
+  <link rel="stylesheet" href="styles/rafting.css">
+  <link rel="preconnect" href="https://fonts.google.com/specimen/Open+Sans">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+  <header>
+    <img class="logo" src="images/logo.jpg" alt="The logo of our rafting company">
+    <nav class="pages">
+      <a id="index" href="index.html">Home</a>
+      <a id="about" href="about.html">About Us</a>
+      <a id="trips" href="trips.html">Trips</a>
+      <a id="contact" href="contact.html">Contact Us</a>
+    </nav>
+  </header>
+  <main>
+    <div class="hero">
+      <h1>Rafting adventure</h1>
+      <article>
+        <p class="mission">"At Rafting Adventures, our purpose is to offer thrilling, safe, and eco-friendly rafting
+          adventures that connect people with nature, promote teamwork, and inspire environmental stewardship."</p>
+        <img class="little-hero" src="images/little-hero.jpg" alt="Happy customers" width="100" height="100">
+      </article>
+    </div>
+    <section class="history">
+      <h2>History</h2>
+      <div class="history-content">
+      <p>Our company was founded by a group of lifelong friends with a passion for rafting and a deep
+        love for nature. Committed to sustainability, they use environmentally responsible practices and support local
+        conservation efforts. Their mission is to create</p>
+      <img class="history-content" id="his-cen" src="images/img7.png">
```

Comment:
Please add the `alt` attribute for this image to fix this html validation error:

![Uploading image.png…]()